infra/prod/_modules/functions/function_op.tf

```diff
@@ -0,0 +1,86 @@
+locals {
+  openid_provider_func_settings = {
+    for s in var.op_func.app_settings :
+    s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${var.product}-kv;SecretName=${s.key_vault_secret_name})" : s.value
+  }
+}
+
+module "openid_provider_func" {
```

Comment:
this should be `op_func`

I think the names used in tf configuration should match the resource names

Reply:
[d20958d](https://github.com/pagopa/io-fims/pull/27/commits/d20958d54d6e2a5264f87e33d9a4430fb2ba1523)